openapi/p4pa-classification.openapi.yaml

```
@@ -204,8 +204,126 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ClassificationErrorDTO'
+  /classifications/{organizationId}/export:
+    get:
+      tags:
+        - classification
```

let's introduce a new Controller to handle data export requests
```suggestion
  /export/organization/{organizationId}/classifications:
    get:
      tags:
        - data-exports
```